virtualfish/virtual.fish

```diff
@@ -442,19 +445,44 @@ function __vf_help --description "Print VirtualFish usage information"
 end
 
 function __vf_globalpackages --description "Toggle global site packages"
-  if set -q VIRTUAL_ENV
-      # use site-packages/.. to avoid ending up in python-wheels
-      pushd $VIRTUAL_ENV/lib/python*/site-packages/..
-      if test -e $VIRTUALFISH_GLOBAL_SITE_PACKAGES_FILE
-        echo "Enabling global site packages"
-        command rm $VIRTUALFISH_GLOBAL_SITE_PACKAGES_FILE
-      else
-        echo "Disabling global site packages"
-        touch $VIRTUALFISH_GLOBAL_SITE_PACKAGES_FILE
-      end
-      popd
-    else
-        echo "No virtualenv is active."
+    if set -q VIRTUAL_ENV
+	pushd $VIRTUAL_ENV
+
+	# if pyvenv.cfg is present, toggle configuration value therein
+	# (https://www.python.org/dev/peps/pep-0405/#isolation-from-system-site-packages)
+	#
+	# otherwise use legacy no-global-site-package.txt file in lib/python*
+
+	if test -e $VIRTUALFISH_VENV_CONFIG_FILE  # PEP 405
+	    # toggle
+	    sed -i '/include-system-site-packages/ {s/true/false/;t;s/false/true/}' $VIRTUALFISH_VENV_CONFIG_FILE
```

Comment:
Probably best to replace all `sed […]` calls with `command sed […]` to prevent potential problems for folks who have aliased that command.

Reply:
Good point. Fixed in 10e29e5.